duckduckgo/iOS
# Update Link Tracking Protection to preserve headers

Task/Issue URL: https://app.asana.com/0/1163321984198618/1202368115363949/f
Tech Design URL:
CC:

**Description**:



**Steps to test this PR**:
1. Clone https://github.com/duckduckgo/privacy-test-pages on `brad/header-drop-test` and serve the root directory locally
2. Navigate to http://localhost:3000/privacy-protections/query-parameters/
3. Ensure the tests pass ok
4. Set a breakpoint in `TabViewController.swift` on `Line 1331`
5. Click a link in the test page
6. In the debug console execute `po navigationAction.request.allHTTPHeaders` then `po newRequest.allHTTPHeaders` and ensure they are the same.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
